.github/workflows/testserver-helios.yml

```diff
@@ -0,0 +1,161 @@
+name: Deploy to Testserver with Helios
+
+on:
+  workflow_dispatch:
+    inputs:
+      HELIOS_TRIGGERED_BY:
```

Comment:
does it make sense to introduce HELIOS here? looks like a strong coupling, but ideally Artemis stays independent of the actual CI provider, so I would prefer HELIOS independent variables and names